control/src/pages/settings/styles/deleteWorkspace.css

```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+*{
```

Comment:
use css module, also change your line 3 to 11, don't set global styles it would affect other pages

Reply:
I will update the pull request but pls I need to understand the change from line 3 to 11